backend/Makefile

```diff
@@ -326,10 +326,17 @@ end2end-test: test-deps
 	$(call run-pytest,end to end tests,-m "end2end")
 .PHONY: end2end-test
 
-test: test-deps
+test: test-deps style-test
 	$(call run-pytest,all tests)
 .PHONY: test
 
+style-test:
+	@echo "${INFO}Running code style checks"
+	${PYTHON} -m pipenv install --dev
+	${PYTHON} -m pipenv run black --check .
```

Comment:
Note: We don't need to add `--line-length 120` in this case since it's configured in `pyproject.toml`.